### DISCUSSION TITLE
BB Group related: Implementing Grouping in IfcAsset and IfcInventory

### DISCUSSION THREAD

**Discussion by frisoBUW** [IfcOpenShell/IfcOpenShell#2307] [Ideas]
So far, there are Options to create IfcGroup and IfcSystem (Subtype). Grouping Elements in IfcAsset or IfcInventory is not supported so far. Implementing these specific Groups would help with Facility Management Use Cases. 
Greetings

**Comment by Moult** [comment]
Can you add this as a bug? Sorry I tend to miss discussions esp. if they are feature requests.
